File: CC++/Temp/New folder/Triangle.java
import java.util.*;
public class Triangle {
    public static void main(String args[]) {
        
        Scanner in = new Scanner(System.in);
        System.out.print("Enter Three Numbers : ");
        double a = in.nextDouble();
        double b = in.nextDouble();
        double c = in.nextDouble();
        
        if(a == b && b==c)
            System.out.println("Equilateral Triange");
        else if(a==b || b==c || c==a)
            System.out.println("Isosceles Triangle");
        else 
            System.out.println("Scalene Triangle");
        
        if(a*a + b*b == c*c || c*c + b*b == a*a || a*a + c*c == b*b)
            System.out.println("Right Angle Triangle");
        

    }
}

File: CC++/Assignments/Assignment 5/Triangle.c
/*Triangle Types and Validity*/

#include <stdio.h>

int main()
{
    float A, B, C;
    printf("Enter sides of Triangle : ");
    scanf("%f%f%f", &A, &B, &C);

    if (((A + B) > C) && ((B + C) > A) && ((C + A) > B))
    {
        if (A == B && B == C)
            printf("Equilateral");
        else if (A == B || B == C || C == A)
            printf("Isosceles");
        else if (((A * A + B * B) == C * C) || ((C * C + B * B) == A * A) || ((C * C + A * A) == B * B))
            printf("Right Angle Triangle");
        else
            printf("Scalene Triangle");
    }
    else
        printf("Not a valid Triangle");

    return 0;
}
